lib/commands/livesync.ts

```diff
@@ -0,0 +1,26 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+export class LivesyncCommand implements ICommand {
+	constructor(private $logger: ILogger,
+		private $usbLiveSyncService: IUsbLiveSyncService,
+		private $mobileHelper: Mobile.IMobileHelper) { }
+
+	public execute(args: string[]): IFuture<void> {
+		return this.$usbLiveSyncService.liveSync(args[0]);
+	}
+	
+	public canExecute(args: string[]): IFuture<boolean> {
+		return (() => {
+			let platform = args[0];
```

Comment:
This way the command will work no matter of the number of arguments passed, for example
`tns livesync android goshko ios peshko john`